shell/browser/web_dialog_helper.cc

```diff
@@ -305,18 +111,19 @@ WebDialogHelper::~WebDialogHelper() = default;
 void WebDialogHelper::RunFileChooser(
     content::RenderFrameHost* render_frame_host,
     scoped_refptr<content::FileSelectListener> listener,
-    const blink::mojom::FileChooserParams& params) {
+    const FileChooserParams& params) {
   file_dialog::DialogSettings settings;
   settings.force_detached = offscreen_;
   settings.filters = GetFileTypesFromAcceptType(params.accept_types);
   settings.parent_window = window_;
   settings.title = base::UTF16ToUTF8(params.title);
 
-  auto file_select_helper = base::MakeRefCounted<FileSelectHelper>(
-      render_frame_host, std::move(listener), params.mode);
+  auto* fsc =
+      new FileSelectHelper(render_frame_host, std::move(listener), params.mode);
```

Comment:
where does `FileSelectHelper` get deleted since it no longer ref counted ?

Reply:
Class should be explicitly deleted wherever `Release` was called previously https://github.com/electron/electron/pull/25030/commits/7c6f600b2424b1fd7d61b17a35394dc7f590728a